Makefile

```diff
@@ -56,8 +56,11 @@ website/build-local:
 .PHONY: license-check
 license-check:
 	go mod vendor
-	licensei check
-	licensei header
+	licensei cache --debug
```

Comment:
I believe you don't need the `cache` command here. Just `check`. Cache command you can run locally to re-generate the cache file, which needs to be committed.

Reply:
I was thinking about ensuring that the cache file is up to date with the last `git diff --exit-code` command.
WDYT?

Reply:
If it's not up-to-date, `check` command will do additional slower checks (HTTP calls). I think the benefit of the `cache` is to speed up the run and avoid hitting API request limits.
In the CI I would personally just put:
```bash
licensei check
licensei header
```
The rest shouldn't be necessary IMO. Still, if you think this is better, go for it.

Reply:
Maybe we should not version the licensei cache file in git but rather store it between jobs in the Github Actions cache:

https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows

https://github.com/actions/cache